docs/license-documentation.rst

```diff
@@ -0,0 +1,75 @@
+
+mi-deploy: `Apache 2.0 <https://github.com/CiscoCloud/microservices-infrastructure/blob/master/LICENSE>`_
+
+Packer: `MPL 2.0 <https://github.com/mitchellh/packer/blob/master/LICENSE>`_
+
+playbooks: `Apache 2.0 <https://github.com/CiscoCloud/microservices-infrastructure/blob/master/LICENSE>`_
+
+plugins/callbacks/profile_tasks.py: `MIT <https://github.com/CiscoCloud/microservices-infrastructure/blob/master/plugins/callbacks/profile_tasks.py>`_
+
+plugins/inventory/terraform.py: `Apache 2.0 <https://github.com/CiscoCloud/microservices-infrastructure/blob/master/plugins/inventory/terraform.py>`_
+
+roles/calico: `Apache 2.0 <https://github.com/projectcalico/calico/blob/master/LICENSE>`_
+
+roles/chronos: `Apache 2.0 <https://github.com/mesos/chronos/blob/master/LICENSE>`_
+
+roles/collectd: `MIT <https://github.com/CSC-IT-Center-for-Science/ansible-role-collectd/blob/master/LICENSE>`_
+
+roles/common: `MIT <https://github.com/sunscrapers/ansible-role-common/blob/master/LICENSE>`_
+
+roles/ consul-template`MPL 2.0 <https://github.com/hashicorp/consul-template/blob/master/LICENSE>`_
+
+roles/consul: `MLP 2.0 <https://github.com/hashicorp/consul/blob/master/LICENSE>`_
+
+roles/dnsmasq: `Apache 2.0 <https://github.com/mesos/chronos/blob/master/LICENSE>`_
+
+roles/docker: `Apache 2.0 <https://github.com/docker/docker/blob/master/LICENSE>`_
+
+roles/etcd: `Apache 2.0 <https://github.com/coreos/etcd/blob/master/LICENSE>`_
+
+roles/flannel: `Apache 2.0 <https://github.com/coreos/flannel/blob/master/LICENSE>`_
+
+roles/glusterfs: `GNU GPL 3.0 <https://github.com/rosmo/ansible-glusterfs/blob/master/LICENSE>`_
+
+roles/handlers: `Apache 2.0 <https://github.com/CiscoCloud/microservices-infrastructure/blob/master/LICENSE>`_
+
+roles/haproxy: `Apache 2.0 <https://github.com/vmware/ansible-role-haproxy/blob/master/LICENSE>`_
+
+roles/kubernetes, k8's: `Apache 2.0 <https://github.com/vmware/ansible-role-kubernetes-master/blob/master/LICENSE>`_
+
+roles/logrotate: `MIT <https://github.com/retr0h/ansible-logrotate/blob/master/LICENSE>`_
+
+roles/logstash: `Apache 2.0 <https://github.com/elastic/logstash/blob/master/LICENSE>`_
+
+roles/lvm: license: `GNU GPL 3.0 <https://github.com/elastic/logstash/blob/master/LICENSE>`_
+
+roles/mantlui: `Apache 2.0 <http://microservices-infrastructure.readthedocs.org/en/latest/license.html>`_
+
+roles/marathon: `Apache 2.0 <https://github.com/mesosphere/marathon/blob/master/LICENSE>`_
+
+roles/mesos: `Apache 2.0 <https://github.com/apache/mesos/blob/master/LICENSE>`_
+
+roles/nginx: `MIT <https://github.com/ANXS/nginx>`_
+
+roles/traefik: `MIT <https://github.com/emilevauge/traefik/blob/master/LICENSE.md>`_
+
+roles/vault: `MLP 2.0 <https://github.com/hashicorp/vault/blob/master/LICENSE>`_
+
+roles/zookeeper: `Apache 2.0 <https://github.com/apache/zookeeper/blob/trunk/LICENSE.txt>`_
+
+ssl: `Apache 2.0 <https://github.com/CiscoCloud/microservices-infrastructure/blob/master/LICENSE>`_
```

Comment:
This can also be removed, as it falls under the Mantl license as part of the project.